packages/vue/src/components/field/field-input.vue

```diff
@@ -8,17 +8,28 @@ export interface FieldInputProps
     /**
      * @vue-ignore
      */
-    InputHTMLAttributes {}
+    Omit<InputHTMLAttributes, 'value'> {
+  modelValue?: InputHTMLAttributes['value']
+}
 </script>
 
 <script setup lang="ts">
 import { ark } from '../factory'
 import { useFieldContext } from './use-field-context'
 
-defineProps<FieldInputProps>()
+const props = defineProps<FieldInputProps>()
```

Comment:
It shouldn't be needed to add the `const props =` since it's only used in the `<template>` and there it knows the props and you can call them through `modelValue` instead of `props.modelValue`.